### PR TITLE
 Update more copyright banners

### PR DESCRIPTION
**Pull request by mwoehlke-kitware**
Update copyright banners in the remainder of arrows/, except for dbow2 (per discussion in #1144), and also in extras/.

See also #1142 and other PRs referencing that.

### CODE REVIEW

File: arrows/core/video_input_filesystem_buffer.h [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+ * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+ * DAMAGE.
+ */
+
+#ifndef ARROWS_CORE_VIDEO_INPUT_IMAGE_LIST_H
```
Comment (mleotta) [inline arrows/core/video_input_filesystem_buffer.h:32]: This looks like an entirely new file.  Does not belong in this PR

File: arrows/serialize/protobuf/detected_object_type.h [hunk 2]
```diff
@@ -1,32 +1,7 @@
...
- * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
- * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
- * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
- * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
  */
```
Comment (mleotta) [inline arrows/serialize/protobuf/detected_object_type.h:5]: Is this a stray comment comment closing?

File: arrows/core/video_input_filesystem_buffer.h (hunk 1 above)
Comment (mwoehlke-kitware) [reply arrows/core/video_input_filesystem_buffer.h]: Ack, that's what I get for using `git diff` + `git add` instead of `git add -p`... good catch, thanks! (Should be fixed now.)

File: arrows/serialize/protobuf/detected_object_type.h (hunk 2 above)
Comment (mwoehlke-kitware) [reply arrows/serialize/protobuf/detected_object_type.h]: Oh, dear... someone added a *second* line of copyright:
```
/*ckwg +29
 * Copyright 2018, 2020 by Kitware, Inc.
 * Copyright 2018-2020 by Kitware, Inc.
```
...and didn't fix the marker, causing the script to delete the "wrong" number of lines. Should be fixed, thanks!

**Review by mleotta — APPROVED** [review]

**Review by dstoup — APPROVED** [review]
I like this change. Code edits and reviews should be greatly simplified as a result. Thanks!

**Comment by dstoup** [comment]
Here is the Centos error

In file included from /home/kitware/jenkins/workspace/KwiverCentOSPR/arrows/serialize/protobuf/activity_type.cxx:5:0:
/home/kitware/jenkins/workspace/KwiverCentOSPR/arrows/serialize/protobuf/activity_type.h:5:3: error: expected unqualified-id before ‘/’ token
  */

**Comment by mwoehlke-kitware** [comment]
I found another instance of a doubled copyright line making the eyecatcher have the wrong number of lines (and resulting in a stray `*/`). Should be fixed now.

**Comment by mwoehlke-kitware** [comment]
...and this, kids, is why it's good to have eyeballs (and ideally, more than one set) on even fully mechanical changes :grinning:.
